agent/runner/jobs/mongodb_restore_job.go

```diff
@@ -180,12 +193,21 @@ func (j *MongoDBRestoreJob) findSnapshot(ctx context.Context) (*pbmSnapshot, err
 			if len(list.Snapshots) == 0 {
 				j.l.Debugf("Try number %d of getting list of artifacts from PBM is failed.", checks)
 				if checks > maxListChecks {
-					return nil, errors.New("failed to find backup entity")
+					return nil, errors.Wrap(ErrNotFound, "got no one snapshot")
```

Comment:
```suggestion
					return nil, errors.Wrap(ErrNotFound, "got no snapshot")
```
It could also be `didn't get a single snapshot`.

Reply:
TBH don't see the difference). Let me leave it as is)

Reply:
Well, it's not really correct from the grammar perspective :)